cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -423,6 +423,15 @@ handleSimpleNode runP p2pMode tracers nc onKernel = do
               , srnMaybeMempoolCapacityOverride = ncMaybeMempoolCapacityOverride nc
               }
       DisabledP2PMode -> do
+#ifdef UNIX
+        _ <- Signals.installHandler
+              Signals.sigHUP
+              (Signals.Catch $ do
+                traceWith (startupTracer tracers)
+                        $ NetworkConfigUpdateError "not supported in non-p2p mode"
```

Comment:
Don't we need to clarify what is not supported? (configuration reloading in this case I guess :) )

Reply:
Yes, and actually we should use a different massage.  This is not an error, just a warning.

Reply:
I changed it to `NetworkConfigUpdateUnsupported` which expands to:
```
Network topology reconfiguration is not supported in non-p2p mode
```